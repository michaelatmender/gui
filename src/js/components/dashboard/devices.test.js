import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { createMount } from '@material-ui/core/test-utils';
import Devices from './devices';

it('renders correctly', () => {
  const tree = createMount()(
    <MemoryRouter>
      <Devices />
    </MemoryRouter>
  );
  expect(tree.html()).toMatchSnapshot();
});
