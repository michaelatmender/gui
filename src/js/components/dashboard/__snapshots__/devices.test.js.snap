// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div><h4 class=\\"dashboard-header\\"><span>Devices</span></h4><div style=\\"margin-bottom: 30px; margin-top: 50px;\\"><div class=\\"notActive widget \\"><div style=\\"height: 100%; width: 100%; display: flex; flex-direction: column;\\"><div style=\\"border-bottom-style: solid; display: flex; flex-direction: row;\\" class=\\"widgetHeader\\"><div style=\\"display: flex; flex-direction: row;\\"><svg class=\\"MuiSvgIcon-root warning\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" role=\\"presentation\\" style=\\"margin-right: 10px; height: 14px; width: 14px;\\"><path d=\\"M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z\\"></path></svg><div style=\\"display: flex; flex-direction: column;\\"><div class=\\"hint\\"> devices may be offline</div><div class=\\"tiny\\">Inactive for past 24h</div></div></div></div><div style=\\"display: flex; flex-direction: column; align-items: flex-end;\\" class=\\"widgetMainContent align-right\\"><div class=\\"header\\">Accepted devices</div><div class=\\"counter\\"></div></div><span class=\\"link\\"></span></div></div><div class=\\"onboard widget\\"><div><p class=\\"muted\\">Learn how to connect a device</p></div><a class=\\"MuiButtonBase-root MuiButton-root MuiButton-text\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\" href=\\"/devices\\"><span class=\\"MuiButton-label\\">Connect a device</span><span class=\\"MuiTouchRipple-root\\"></span></a></div></div></div>"`;
